new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/StdModule.php

```diff
@@ -269,6 +269,23 @@ public function addConfiguration($key, $value, $groupId, $sortOrder, $setFunctio
         xtc_db_query("INSERT INTO `" . TABLE_CONFIGURATION . "` (`configuration_key`, `configuration_value`, `configuration_group_id`, `sort_order`, `set_function`, `use_function`, `date_added`) VALUES ('$key', '$value', '$groupId', '$sortOrder', '$setFunction', '$useFunction', NOW())");
     }
 
+    public function removeConfigurationAll(): bool
+    {
+        $module_name          = $this->getModulePrefix();
+        $remove_configuration = xtc_db_query(
+            sprintf(
+                /** TRANSLATORS: %1$s: Database table "configuration". %2$s: Value for "configuration_key". */
+                'DELETE FROM `%1$s` WHERE `configuration_key` LIKE "%2$s_%"',
+                TABLE_CONFIGURATION,
+                $module_name
+            )
+        );
+
+        $success = false !== $remove_configuration;
```

Comment:
That works, but could you perhaps write it as an `if` or as a `?` expression? I think then you can read the code a little faster/better. :-)